resources/check-ts.js

```diff
@@ -0,0 +1,19 @@
+// @noflow
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const { exec } = require('./utils');
+
+const source = 'tsconfig.json';
+const destination = path.join('./src', 'tsconfig.json');
+
+fs.copyFileSync(source, destination);
+
+try {
+  exec('dtslint src', { stdio: 'inherit' });
+} finally {
+  fs.unlinkSync(destination);
```

Comment:
Not an issue for me since I use Vim but flickering files create an issues for IDE users.